ocaml/idl/datamodel_common.ml

```diff
@@ -90,7 +90,7 @@ let lima_release_schema_major_vsn = 5
 let lima_release_schema_minor_vsn = 203
 
 let naples_release_schema_major_vsn = 5
-let naples_release_schema_minor_vsn = 400
+let naples_release_schema_minor_vsn = 401
```

Comment:
Don't change this: it is just documenting the schema version of the (already released) Naples release.